### PR TITLE
Adding two-day reminder email template

### PR DESCRIPTION
**Pull request by ns-rse**
Additional email template for when the course is run in-person over two days.

### CODE REVIEW

File: emails/joining_reminder_email_two_day.Rmd [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    lead_instructor: "Neil Shephard"
+---
+
+
+<!--- Send this email the morning of the event --->
+
+We're looking forward to welcoming you to the RSE Sheffield training session on **git & GitHub through GitKraken
+ Client - from Zero to Hero** shortly!
+
+Todays session
```
Comment (tdjames1) [inline emails/joining_reminder_email_two_day.Rmd:29]:
```suggestion
Today's session
```

File: emails/joining_reminder_email_two_day.Rmd [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+(`r format(strptime(params$day1, format = "%Y-%m-%d"), "%a %b %d %Y")`) starts at **`r params$start_time_day1`** in
+**`r params$venue_day1`**. Tomorrow session (`r format(strptime(params$day2, format = "%Y-%m-%d"), "%a %b %d %Y")`)
```
Comment (tdjames1) [inline emails/joining_reminder_email_two_day.Rmd:31]:
```suggestion
**`r params$venue_day1`**. Tomorrow's session (`r format(strptime(params$day2, format = "%Y-%m-%d"), "%a %b %d %Y")`)
```

File: emails/joining_reminder_email_two_day.Rmd [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+will begin at **`r params$start_time_day2`**  in **`r params$venue_day2`**.
+
+See you all soon!
+
+Mr `r params$lead_instructor`
```
Comment (tdjames1) [inline emails/joining_reminder_email_two_day.Rmd:36]: "Mr" may not be the appropriate title for the lead instructor.

Comment (ns-rse) [reply emails/joining_reminder_email_two_day.Rmd:36]: Yes, but neither may `Dr` as it has been in other templates (e.g. I don't hold a doctorate, so always had to change it).

What about leaving title out completely and having it as whatever people put in as the value and expecting people to use their common sense?

Comment (tdjames1) [reply emails/joining_reminder_email_two_day.Rmd:36]: Yes, that's probably the most inclusive option.

**Review by tdjames1 — APPROVED** [review]

**Pull request merged**